content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -99,10 +99,13 @@ limitation and compatibility rules will change.
 
 #### Pause Image
 
-Microsoft maintains a Windows pause infrastructure container at
-`mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
-Kubernetes maintains a multi-architecture image `k8s.gcr.io/pause:3.5` that
-supports Linux as well as Windows.
+Kubernetes maintains a multi-architecture image that includes support for Windows at `k8s.gcr.io/pause:3.5`.
+The [source code](https://github.com/kubernetes/kubernetes/tree/master/build/pause)
+is available on GitHub.
+
+Microsoft maintains a multi-architecture image with Linux and Windows amd64 support at `mcr.microsoft.com/oss/kubernetes/pause:3.5`.
+This image is built from the same source as the Kubernetes maintained image but all of the Windows binaries are [authenticode signed](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/authenticode) by Microsoft.
+For this reason the Microsoft maintained image is recommended for production environments.
```

Comment:
```suggestion
The Microsoft maintained image is recommended for production environments when signed binaries are required
```

using "this" was a little confusing.

Reply:
Can you add the trailing . into you suggestion?
If you do that I'll commit the changes.

Reply:
done